docs/source/algorithms/MayersSampleCorrection-v1.rst

```diff
@@ -97,7 +97,7 @@ References
 .. [1] Lindley, E.J., & Mayers, J. Cywinski, R. (Ed.). (1988). Experimental method and corrections to data. United Kingdom: Adam Hilger. - https://inis.iaea.org/search/search.aspx?orig_q=RN:20000574
 
 
-.. seealso :: Algorithm :ref:`algm-MultipleScatteringCylinderAbsorption`
```

Comment:
This shouldn't be necessary to add any more as the `seeAlso()` method of the algorithm should insert it into the documentation.